common/errors/ATTRIBUTE_MISMATCH_ERROR.go

```diff
@@ -0,0 +1,9 @@
+package errors
+
+import "net/http"
+
+// Used when the user attempts to perform an action that is not permitted as part of the flow.
+// E.g. Attempting to check-in without having RSVPed.
+func ATTRIBUTE_MISMATCH_ERROR(message string) APIError {
+	return APIError{Status: http.StatusUnprocessableEntity, Type: "AttributeMismatchError", Message: message}
```

Comment:
I think my earlier comment was unclear. I was saying the `Type` attribute should be `ATTRIBUTE_MISMATCH_ERROR`. The function name should still be `AttributeMismatchError`.

Reply:
Got it. Besides this, do you have anything else to say on the error choices / messages etc.?